### PR TITLE
[new release] zmq, zmq-lwt and zmq-async (5.2.1)

### PR DESCRIPTION
**Pull request by andersfugmann**
OCaml bindings for ZeroMQ 4.x

- Project page: <a href="https://github.com/issuu/ocaml-zmq">https://github.com/issuu/ocaml-zmq</a>

##### CHANGES:

* Forward compatibility with Lwt 6.0 (issuu/ocaml-zmq#122, @andersfugmann)
* Fix wrong fd type for zmq-async, effecting s390x (issuu/ocaml-zmq#123, @andersfugmann)

### CODE REVIEW

File: packages/zmq-async/zmq-async.5.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+license: "MIT"
+homepage: "https://github.com/issuu/ocaml-zmq"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.04.1"}
+  "zmq" {= version}
+  "async_unix" {>= "v0.11.0" & < "v0.16"}
+  "async_kernel" {>= "v0.11.0" & < "v0.16"}
+  "base" {>= "v0.11.0" & < "v0.16"}
```
Comment (kit-ty-kate) [inline packages/zmq-async/zmq-async.5.2.1/opam:14]: Is there something preventing it to be not available on v0.16?
```suggestion
  "async_unix" {>= "v0.11.0"}
  "async_kernel" {>= "v0.11.0"}
  "base" {>= "v0.11.0"}
```

**Comment by andersfugmann** [comment]
All tests completed successfully except `compilers,4.14,zmq.5.2.1,revdeps,frama-c.24.0` which failes due to a faulty expect test, and not related to this release. Error copied below:
```
# Log of tests/misc/result/audit-out.json:
# --- tests/misc/oracle/audit-out.json	2021-11-30 00:00:00.000000000 +0000
# +++ tests/misc/result/audit-out.json	2022-11-01 17:27:31.593520744 +0000
# @@ -73,7 +73,6 @@
#    "sources": {
#      "tests/misc/audit.c": "f94ce1c8f5e5911260931783495e2c88",
#      "tests/misc/audit_included.h": "c2cc488143a476f69cf2ed04c3439e6e",
# -    "tests/misc/audit_included_but_not_listed.h":
# -      "c2cc488143a476f69cf2ed04c3439e6e"
# +    "tests/misc/audit_included_but_not_listed.h": "c2cc488143a476f69cf2ed04c3439e6e"
#    }
#  }

```

File: packages/zmq-async/zmq-async.5.2.1/opam (hunk 1 above)
Comment (andersfugmann) [reply packages/zmq-async/zmq-async.5.2.1/opam:14]: No. I will remove the upper bounds.

**Comment by Leonidas-from-XIV** [comment]
Yeah, looks like the frame-c test is just fragile and should compare the output using JSON equality, not text equality.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**